src/components/AttemptsCounter.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+interface Props {
+  retryLimit: number
+  retriesAllowed: number
+}
+
+export const AttemptsCounter: React.FC<Props> = ({ retryLimit, retriesAllowed }) => (
```

Comment:
Super minor: This declaration is somewhat inconsistent in this code base.

Reply:
This one is probably extremely obvious, but it is going over my head. What is inconsistent about the declaration?

Reply:
Ah -- just the use of `React.FC<Props>` to define the type. It might be "more" consistent as:

```javascript
export const AttemptsCounter = ({ retryLimit, retriesAllowed }: Props): JSX.Element => (
```

Reply:
That makes sense. I'll revise accordingly.

Reply:
Updated per this [commit](https://github.com/openstax/k12-apps-raise/pull/89/commits/040f8931d9c2d6451b0d23a1920bccde6a4843cd).